core/src/avm2/globals/xml.rs

```diff
@@ -147,6 +149,19 @@ pub fn children<'gc>(
     Ok(XmlListObject::new(activation, children, Some(xml.into())).into())
 }
 
+pub fn parent<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let xml = this.unwrap().as_xml_object().unwrap();
+    let node = xml.node();
+    if node.parent().is_none() {
```

Comment:
Using is_none and unwrap doesn't feel very rusty to me. You can probably use Option::map_or here.

Reply:
should be resolved now!